index.js

```diff
@@ -1,3 +1,9 @@
+var assign = require('object-assign');
```

Comment:
It's in the `package.json` now.

I too was a bit conflicted on this.  My rationale was that since this module only exports as CommonJS, any consumer will need a module bundler too in order to use it anyway, which will pick up the dependency too.  So I don't think you need to do any bundling on your end.